### ISSUE TITLE
ArchiveFileName inside compressed zip

### ISSUE THREAD

**Issue by eshref** [NLog/NLog#4647]
Hi!

Currently I have below target:

```
<target name="targetName" xsi:type="File" fileName="logs\someservice.log"
            archiveAboveSize="1024"
            maxArchiveDays="3"
            archiveNumbering="DateAndSequence"
            archiveEvery="Day"
            enableArchiveFileCompression="true">
```

When archiving period is reached nlog creates [someservice.20211102.0.zip] files and so on. It is ok. But inside this zip file archived log file name is [someservice.log]. I want it to be [someservice.20211102.0.log]. How can I achieve this?

**Comment by snakefoot** [comment]
Think it has been resolved with #4189 and available with [NLog 5.0](https://nlog-project.org/2021/08/25/nlog-5-0-preview1-ready.html) (Current [Preview 3](https://www.nuget.org/packages/NLog/5.0.0-preview.3))

**Comment by snakefoot** [comment]
Closing issue as resolved by NLog 5.0
